cli/crates/cli/tests/utils/environment.rs

```diff
@@ -137,7 +149,7 @@ impl Environment {
         let command = cmd!(
             cargo_bin("grafbase"),
             "--trace",
-            "2",
+            "3",
```

Comment:
What is this for?  We only have levels 0,1,2 AFAIK.

Reply:
Yeah, sorry, mistake